tls/Network/TLS/Core.hs

```diff
@@ -112,7 +112,7 @@ bye ctx = liftIO $ do
                 unless recvNST $ do
                     rtt <- getRTT ctx
                     var <- newEmptyMVar
-                    _ <- forkIOWithUnmask $ \umask ->
+                    _ <- forkIOWithUnmask $ \umask -> bestEffort $
```

Comment:
For me, `E.handle bestEffort` is more readable than `bestEffort` with `E.handle` removed from the definition of `bestEffort`.